frontend/src/index.css

```
@@ -120,3 +120,14 @@
     @apply bg-background text-foreground;
   }
 }
+
+@layer utilities {
+  .bg-grid-white\/\[0\.05\] {
+    background-image: linear-gradient(
+        to right,
+        rgba(255, 255, 255, 0.05) 1px,
```

Stylelint (17.2.0)

[error] Expected "0.05" to be "5%" (alpha-value-notation)

(alpha-value-notation)